source/uwp/AdaptiveCards.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
>Microsoft [](start = 0, length = 9)

Don't include this file

Reply:
will do

---
In reply to: [157614875](https://github.com/Microsoft/AdaptiveCards/pull/1045#discussion_r157614875) [](ancestors = 157614875)